src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -1474,24 +1474,25 @@ public void requiredTurnSeries(final PlayerID player) {
     if (player == null) {
       return;
     }
-    ThreadUtil.sleep(300);
-    SwingAction.invokeAndWait(() -> {
-      final Boolean play = requiredTurnSeries.get(player);
-      if (play != null && play) {
-        ClipPlayer.play(SoundPath.CLIP_REQUIRED_YOUR_TURN_SERIES, player);
-        requiredTurnSeries.put(player, false);
-      }
-      // center on capital of player, if it is a new player
-      if (!player.equals(lastStepPlayer)) {
-        lastStepPlayer = player;
-        data.acquireReadLock();
-        try {
-          mapPanel.centerOn(TerritoryAttachment.getFirstOwnedCapitalOrFirstUnownedCapital(player, data));
-        } finally {
-          data.releaseReadLock();
+    if(ThreadUtil.sleep(300)) {
```

Comment:
Would it be better to just return from the method if return false rather than nesting this?